src/plugins/intel_gpu/src/kernel_selector/cl_kernels/reorder_data.cl

```diff
@@ -132,7 +132,11 @@ KERNEL (reorder_data)(
     float Ycomponent = mad(Y.x, 296.82f, -18.624f);
     float res = clamp(Ycomponent, 0.f, 255.f);
 #else
-    CALC_TYPE res = TO_CALC_TYPE(input[input_idx]);
+    #ifdef BF16_INPUT
+        CALC_TYPE res = _convert_as_bfloat16_float(input[input_idx]);
```

Comment:
isn't it necessary to have additional macro to convert type to CALC_TYPE?
I mean, `CALC_TYPE res = TO_CALC_TYPE(_convert_as_bfloat16_float(input[input_idx]));`

Reply:
done, added macro